enderio-conduits/src/main/java/crazypants/enderio/conduit/packet/PacketConnectionMode.java

```diff
@@ -14,15 +14,15 @@
 public class PacketConnectionMode extends AbstractConduitPacket<IConduit> {
 
   private EnumFacing dir;
-  private boolean next;
```

Comment:
and this file is a 100% undo of my commit

Reply:
Yup I cleaned up this code but forgot to commit it

Reply:
fixed it now, the new version takes a mode instead of a boolean for the next input.